pkg/security/config/config.go

```diff
@@ -245,6 +245,25 @@ func NewConfig() (*Config, error) {
 func NewRuntimeSecurityConfig() (*RuntimeSecurityConfig, error) {
 	sysconfig.Adjust(coreconfig.SystemProbe)
 
+	eventTypeStrings := map[string]model.EventType{}
```

Comment:
Do you think we could move this into a function ? I understand the goal of removing the global, but maybe you could move all this into a function ?

Reply:
This function is called multiple times. So we could build create 2 functions:
- one that creates `eventTypeStrings`
- pass it to `parseEventTypeStringSlice`